sdk/cognitivelanguage/ai-language-text/test/public/expectations.ts

```diff
@@ -2184,66 +2184,66 @@ export const expectation25: AnalyzeBatchResult[] = [
                   text: "Pathology study",
                 },
                 subject: {
-                  reference: "Patient/08753679-6896-4364-88cb-1a6ab88161ee",
+                  reference: "Patient/9b180042-487d-4f04-aec8-b63dede967a9",
                   type: "Patient",
                 },
                 encounter: {
-                  reference: "Encounter/272b0e3f-a564-45f8-8c0d-125eb30c7315",
+                  reference: "Encounter/46a049bd-2b0e-4325-8025-60c2ddb4f8a1",
                   type: "Encounter",
                   display: "unknown",
                 },
                 date: "0001-01-01T00:00:00+00:00",
                 author: [
                   {
-                    reference: "Practitioner/238c9e8e-512e-4d6a-a21b-86a6a130b2fa",
+                    reference: "Practitioner/c4998788-8e0e-454f-a583-ddc2aa60d81c",
                     type: "Practitioner",
                     display: "Unknown",
                   },
                 ],
                 title: "Pathology study",
                 section: [
                   {
-                    title: "General",
+                    title: "(General)",
```

Comment:
Hi @gils-ms, is this change (the parenthesis around the title) in FHIR output expected?